frontend/reducers/user.js

```diff
@@ -5,6 +5,10 @@ import {
 export default function user(state = false, action) {
   switch (action.type) {
   case USER_RECEIVED:
+    if (!action.user) {
+      return false
```

Comment:
TIL `false.foo` is something that can actually have a useful meaning. We're getting a little lucky here, but it works.